### PR TITLE
Enable xlpCMLTests in cmdLineTests

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Enable xlpCMLTests in cmdLineTests
- issue: runtimes/backlog 374

[ci skip]
Signed-off-by: Longyu Zhang <longyu.zhang@ibm.com>

### CODE REVIEW

**Comment by LongyuZhang** [comment]
Grinder Links:

ibm:
xlinux_xl 10281   xlinux 10303    win 10301  win_xl 10302  aix 10295   aix_xl 10296   plinux 10307    plinux_xl 10305   zos 10300   zos_xl 10283   zlinux 10298   zlinux_xl  10284

openj9_8:
xlinux_xl 10280    xlinux  10287   win 10288   win_xl 10289   aix 10290   plinux_xl 10282   zlinux 10292   zlinux_xl  10286   mac 10293    mac_xl 10285

openj9_11:
xlinux_xl 10306   xlinux 10311   win_xl 10310   plinux_xl 10309   mac_xl 10308

**Comment by LongyuZhang** [comment]
@llxia Could you review it? Thanks.

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

File: test/functional/cmdLineTests/xlpCMLTests/playlist.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2018, 2020 IBM Corp. and others
```
Comment (llxia) [inline test/functional/cmdLineTests/xlpCMLTests/playlist.xml:3]: This is a new file, so it should be 2020, 2020

File: test/functional/cmdLineTests/xlpCMLTests/build.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+
+<!--
+  Copyright (c) 2018, 2020 IBM Corp. and others
```
Comment (llxia) [inline test/functional/cmdLineTests/xlpCMLTests/build.xml:4]: This is a new file, so it should be 2020, 2020

File: test/functional/cmdLineTests/xlpCMLTests/playlist.xml (hunk 1 above)
Comment (LongyuZhang) [reply test/functional/cmdLineTests/xlpCMLTests/playlist.xml]: Updated to 2020, 2020. Thanks.

File: test/functional/cmdLineTests/xlpCMLTests/build.xml (hunk 2 above)
Comment (LongyuZhang) [reply test/functional/cmdLineTests/xlpCMLTests/build.xml]: Updated to 2020, 2020. Thanks.

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**